diff --git a/src/OrcaSlicer.cpp b/src/OrcaSlicer.cpp
index e3e0a5e..d192d86 100644
--- a/src/OrcaSlicer.cpp
+++ b/src/OrcaSlicer.cpp
@@ -1184,7 +1184,7 @@ int CLI::run(int argc, char **argv)
     // On Linux, wxGTK has no support for Wayland, and the app crashes on
     // startup if gtk3 is used. This env var has to be set explicitly to
     // instruct the window manager to fall back to X server mode.
-    ::setenv("GDK_BACKEND", "x11", /* replace */ true);
+    ::setenv("GDK_BACKEND", "wayland", /* replace */ true);
 
     // Also on Linux, we need to tell Xlib that we will be using threads,
     // lest we crash when we fire up GStreamer.
