diff --git a/src/main/settings.ts b/src/main/settings.ts
index b2aeea9..58d0cd9 100644
--- a/src/main/settings.ts
+++ b/src/main/settings.ts
@@ -27,8 +27,10 @@ function loadSettings<T extends object = any>(file: string, name: string) {
 
     const store = new SettingsStore(settings);
     store.addGlobalChangeListener(o => {
-        mkdirSync(dirname(file), { recursive: true });
-        writeFileSync(file, JSON.stringify(o, null, 4));
+        try {
+          mkdirSync(dirname(file), { recursive: true });
+          writeFileSync(file, JSON.stringify(o, null, 4));
+        } catch {}
     });
 
     return store;
