diff --git a/packages/opencode/src/session/system.ts b/packages/opencode/src/session/system.ts
index 399cad8c..33c2d94d 100644
--- a/packages/opencode/src/session/system.ts
+++ b/packages/opencode/src/session/system.ts
@@ -30,7 +30,7 @@ export namespace SystemPrompt {
     if (modelID.includes("gemini-")) return [PROMPT_GEMINI]
     if (modelID.includes("claude")) return [PROMPT_ANTHROPIC]
     if (modelID.includes("polaris-alpha")) return [PROMPT_POLARIS]
-    return [PROMPT_ANTHROPIC_WITHOUT_TODO]
+    return [PROMPT_ANTHROPIC]
   }
 
   export async function environment() {
